charts/nginx-ingress/Chart.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: v2
 name: nginx-ingress
 version: 1.0.2
```

Comment:
The helm chart version should be updated

Reply:
Should be updated now

Reply:
It looks like the current automation updates only the Ingress Controller version for our docs, so anything helm related is missed. This was why `using-the-jwt-token-docker-secret.md` wasn't updated.

I'm not yet sure why the version of Helm in Chart.yaml wasn't updated and the version of IC was.

Reply:
I can still find some 1.0.2's in the branch. maybe we should run the script again locally?
https://github.com/nginxinc/kubernetes-ingress/blob/docs/release-3.4.0/charts/nginx-ingress/README.md?plain=1#L82
https://github.com/nginxinc/kubernetes-ingress/blob/docs/release-3.4.0/charts/nginx-ingress/README.md?plain=1#L89
https://github.com/nginxinc/kubernetes-ingress/blob/docs/release-3.4.0/docs/content/installation/installing-nic/installation-with-helm.md?plain=1#L69
more in the same file...

Reply:
I'm not sure how to see the logs of this script? Maybe we can look at the logs and find out why the helm version is not updated. https://github.com/nginxinc/kubernetes-ingress/blob/docs/release-3.4.0/.github/scripts/release-version-update.sh